### DISCUSSION TITLE
method for cdist() with partial_ratio_alignment

### DISCUSSION THREAD

**Discussion by hk528** [rapidfuzz/RapidFuzz#355] [Q&A]
I want to get "partial_ratio_alignment" between string lists.
If I run "cdist(list_a, list_b, scorer=partial_ratio)", I get only scores. I want to get at least the "dest_start" values.

I think I need to make the return value as float or int. Is it possible to implement "scorer=patial_ratio_alingment.dest_start" ?

RapidFuzz is very useful. Thank you maxbachmann!
This is my first time asking a question on github. I hope this is not a misguided question.

**Comment by maxbachmann** [comment]
I do not think I will add something like this in, but if it's important to you you can absolutely write an extension doing something like this. The large question is whether you need it from a performance perspective:
```python
for a in list_a:
    for b in list_b:
        partial_ratio_alignment(a, b)
```
is too slow, hwile:
```python
process.cdist(list_a, list_b, scorer=partial_ratio)
```
is fast enough.

**Reply by hk528** [reply]
That's right. The method using the for loop is very slow.
I need an HPC environment to apply it to the problem I am working on.
I would like to be able to use a CAD machine to work on the problem.

Many people, including myself, are impressed with the speed of rapidfuzz.
I suggested that we get alignment data by array because I thought it would be beneficial to many people.

If it is technically difficult, I will give up, but I thought it could be handled by changing the data structure without changing the way I have been doing things.
